src/styles/blocks/card.scss

```diff
@@ -0,0 +1,64 @@
+.card {
+  display: flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: all 300ms;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.2);
+  }
+
+  &-image {
+    &__model {
+      display: flex;
+      width: 160px;
+      height: 134px;
+      margin: 0 auto 40px;
+    }
+
+    &__info {
+      color: $main-accent;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      margin-bottom: 0;
+
+      &:hover {
+        color: $hover-color;
+        transition: 300ms;
+      }
+    }
+
+    &__code {
+      margin-top: 4px;
+      margin-bottom: 16px;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
```

Comment:
Default value, you can just remove it everywhere
```suggestion
```

Reply:
I can delete only font-style, I need weight tho